support-frontend/assets/components/spreadTheWord/spreadTheWord.scss

```diff
@@ -1,62 +1,20 @@
 .component-spread-the-word {
+  padding: 0 10px ($gu-v-spacing * 2);
+  margin: 0 -10px;
+  border-bottom: 1px solid gu-colour(neutral-86);
+}
 
-  .component-page-section {
-    border-top: 1px solid gu-colour(neutral-86);
-  }
-
-  .component-page-section__heading {
-    font-family: $gu-headline;
-    font-size: 24px;
-    line-height: 1.17;
-    font-weight: bold;
-  }
-
-  .component-page-section__header {
-    position: relative;
-    padding-right: $gu-h-spacing;
-    box-sizing: border-box;
-
-    @include dropline;
-  }
-
-  .component-page-section__body {
-    position: relative;
-    padding-top: $gu-v-spacing / 2;
-    padding-bottom: $gu-v-spacing * 4;
-
-    @include mq($from: desktop) {
-      padding-left: $gu-h-spacing;
-    }
-  }
-
-  .component-social-share {
-    margin-right: 26px;
-  }
-
-  .svg-facebook, .svg-twitter {
-    background-color: gu-colour(neutral-7);
-    fill: #fff;
-    border: none;
-  }
-
-  .svg-facebook {
-    height: 34px;
-    width: 34px;
-    padding: 10px;
-  }
-
-  .svg-twitter {
-    padding: 13px;
-    height: 28px;
-    width: 28px;
-  }
-
+.component-spread-the-word__title {
+  font-family: $gu-text-egyptian-web;
+  letter-spacing: -0.5px;
+  font-weight: 900;
+  font-size: 17px;
+  margin: ($gu-v-spacing / 3) 0;
 }
 
 .component-spread-the-word__description {
-  margin-bottom: $gu-v-spacing * 2;
-
-  @include mq($from: phablet) {
-    width: 500px;
-  }
+  font-family: $gu-text-egyptian-web;
```

Comment:
can we try `@include gu-fontset-body;` or `@include gu-fontset-body-large;` for this one? cc @ionamckendrick

Reply:
Adding a "refactor typeface styles using mixins" to TBD